src/python/bot/tasks/task_creation.py

```diff
@@ -150,6 +150,9 @@ def create_variant_tasks_if_needed(testcase):
     return
 
   testcase_id = testcase.key.id()
+  testcase_platform = data_types.Job.query(
```

Comment:
testcase.platform has this info and avoids the db call.

Reply:
It seems like testcase platform loses some of the platform id information, hence this call. Thinking about it, that might be fine in the context of gestures, though. Will remove this.

Reply:
No longer relevant in the new implementaiton.